sessions/react-effects-and-localstorage/journal-app-localstorage/src/components/EntriesSection/index.js

```diff
@@ -0,0 +1,46 @@
+import "./EntriesSection.css";
+import Divider from "../Divider";
+import Entry from "../Entry";
+import Tabs from "../Tabs";
+import Tab from "../Tab";
+import Badge from "../Badge";
+import { Fragment } from "react";
+
+export default function EntriesSection({
+  entries,
+  onToggleFavorite,
+  showFavorites,
+  amountAllEntries,
+  amountFavoriteEntries,
+  onShowAllEntries,
+  onShowFavoriteEntries,
+}) {
+  return (
+    <section className="entries-section">
+      <Tabs>
+        <Tab onClick={onShowAllEntries} active={!showFavorites}>
+          All Entries <Badge active={!showFavorites}>{amountAllEntries}</Badge>
+        </Tab>
+        <Tab onClick={onShowFavoriteEntries} active={showFavorites}>
+          Favorites{" "}
+          <Badge active={showFavorites}>{amountFavoriteEntries}</Badge>
+        </Tab>
```

Comment:
```suggestion
        <Tab onClick={onShowFavoriteEntries} active={filter === 'favorites'}>
          Favorites{" "}
          <Badge active={filter === 'favorites'}>{amountFavoriteEntries}</Badge>
        </Tab>
```

Reply:
All in all it doesn't look too bad to me… on the `<Tab>`s i find it even easier to read. Idk though - you decide :)